config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "users#index"
+
+  resources :users do
+    resources :posts
```

Comment:
- [ ] please make sure to avoid unnecessarily exposing endpoints.

Reply:
Hi, @mikethreels 👋 thanks for the review 👏

Am trying to understand your comments and the only thing that comes to my mind is hiding or removing the line `root "users#index"` is that correct ?? if not kindly provide some information or a link that will help me understand your point.

Once again thank you very much ❤